Assignment2/MMN44-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,32 @@
+{
+  "@context": {
+    "vocab": "http:www.example.org/ontology#",
+    "name": "vocab:name",
+    "hometown": "vocab:hometown",
+    "age": "vocab:age",
+    "Alma mater": "vocab:Alma mater",
+    "parent's name": "vocab: parentsname",
+    "subjects": "vocab:subjects",
+    "sname": "subjects:name",
+    "sdesc": "subjects:description",
+    "steach": "subjects:teachers"
```

Comment:
These subject attribute definitions work, but semantically they aren't ok. The best way to define them might be defining a new prefix for the subject concept and then use it to define their attributes.
You don't need to change them, but remember to provide good definitions for the future.